### ISSUE TITLE
 Class 'League\Flysystem\InMemory\InMemoryFilesystemAdapter' not found 

### ISSUE THREAD

**Issue by onet4** [elazar/flystream#1]
Hello,

I just tried the example in readme and it seems 

```php
use League\Flysystem\InMemory\InMemoryFilesystemAdapter;

$adapter = new InMemoryFilesystemAdapter();
```
causes the error `Fatal error: Uncaught Error: Class 'League\Flysystem\InMemory\InMemoryFilesystemAdapter' not found`. 

I could not find the declaration for `InMemoryFilesystemAdapter` in the `vendor` directory downloaded with Composer. 

I'm I missing something?

**Comment by onet4** [comment]
It seems 

```bash
composer require league/flysystem-memory:^3.0
```
is required.

**Comment by elazar** [comment]
@onet4 That's correct. Please review #3. Does this clarify your issue?

**Comment by elazar** [comment]
Closed by #3.

**Comment by elazar** [comment]
@onet4 This change has been released in 0.4.0. Thank you again for your report and your review of the change!
